src/Voron/Impl/Scratch/ScratchBufferPool.cs

```diff
@@ -206,8 +206,17 @@ public PageFromScratchBuffer Allocate(LowLevelTransaction tx, int numberOfPages)
 
                     createNextFile = true;
                 }
+                else if (_options.SupportLargeTransactions)
```

Comment:
This should be the default, but we need to have a discussion on this to make sure we have a coherent strategy here.